app/src/main/java/com/keylesspalace/tusky/viewdata/StatusViewData.kt

```diff
@@ -25,41 +25,63 @@ import com.keylesspalace.tusky.util.shouldTrimStatus
 /**
  * Created by charlag on 11/07/2017.
  *
- * Class to represent data required to display either a notification or a placeholder.
+ * Class to represent data required to display either a status or a placeholder ("Load More" bar).
  * It is either a [StatusViewData.Concrete] or a [StatusViewData.Placeholder].
+ * Can be created either from a ConversationStatusEntity, or by helpers in ViewDataUtils.
```

Comment:
Would you be OK with holding off on adding the comments here, and adding them as suggestions to https://github.com/tuskyapp/Tusky/pull/3436/files#diff-eb35dc96f92845f7f47b51230940e1c57928b5337d164e64567f3511b19b9647 instead? It'll make syncing that PR with head easier, and that PR removes the `.Placeholder` type, so there's probably no point documenting it here.

That PR also includes some of the same comments that you're making here.

Reply:
I guess my question is, is PR #3436 on track for v24 inclusion? If so, then I agree it would make more sense to glom onto 3436. If not, I think it would be better to do a comments-only patch now. I could do the patch-3436 merge conflict resolution myself if that would help.

Reply:
> I guess my question is, is PR https://github.com/tuskyapp/Tusky/pull/3436 on track for v24 inclusion?

¯\_(ツ)_/¯

Needs a code review and more people to try the APK.